src/aerospike_clj/client.clj

```diff
@@ -324,6 +324,17 @@
                  (put db index set-name payload expiration conf))
                (map vector indices set-names payloads expirations)))))
 
+(defn set
+  "`put` with a update policy"
```

Comment:
I think that now that we have quite a few shorthands like `replace-only`, `create` and `update` it might be usefult to link the java documentation for [RecordExistsAction](https://www.aerospike.com/apidocs/java/com/aerospike/client/policy/RecordExistsAction.html). Do you agree?